feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml

```diff
@@ -11,78 +11,125 @@
+                app:textStartPadding="@dimen/session_bottom_sheet_filter_text_padding"
+        />
 
         <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/sessions_recycler"
-            android:orientation="vertical"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            android:layout_marginTop="16dp"
-            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/sessions_bottom_sheet_title"
-            />
+                android:id="@+id/sessions_recycler"
+                android:orientation="vertical"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_marginTop="16dp"
+                app:visibleGone="@{!shouldShowEmptyStateView}"
+                app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/sessions_bottom_sheet_title"
+        />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/sessions_empty_state_view"
+                android:orientation="vertical"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_marginTop="16dp"
+                app:visibleGone="@{shouldShowEmptyStateView}"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/sessions_bottom_sheet_title">
+
+            <ImageView
+                    android:id="@+id/sessions_favorite_image"
+                    android:layout_width="80dp"
+                    android:layout_height="80dp"
+                    android:contentDescription="@string/session_description_favorite_image"
+                    android:scaleType="fitCenter"
+                    android:src="@drawable/ic_bookmark_black_24dp"
+                    android:tint="@color/gray3"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintVertical_chainStyle="packed"
+                    app:layout_constraintBottom_toTopOf="@+id/sessions_empty_state_text"
+                    android:layout_marginBottom="10dp"
+                    app:layout_constraintVertical_bias="0.42"/>
```

Comment:
It seems this experimental value depends on the height of the test device, but it looks okay for now. 👍

Reply:
0.42 | 0.5 | design
:--: | :--: | :--:
<img src=https://user-images.githubusercontent.com/8403570/50904104-ca022700-1462-11e9-90ec-b444aab1f6c6.png width="200" />  | <img src=https://user-images.githubusercontent.com/8403570/50903622-a38fbc00-1461-11e9-9cee-480d714aa288.png width="200" /> | <img src=https://user-images.githubusercontent.com/5106629/50738121-068c1380-1214-11e9-8987-b5e108ca4ba2.png width="200" />

Maybe the reason was the ConstraintLayout had unintentional margin (top=16dp)... 😓 
I removed the margin and place side by side and rethink... and it might be better to set default(0.5). Sorry for the confusion.

Reply:
Np! Your notice made the layout structure better. 👍